generic/service_check.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Santhosh G <santhog4@linux.vnet.ibm.com>
+#
+# Copyright: 2014 Red Hat Inc.
+# Besed on the Sample Idea from:
+# https://github.com/autotest/virt-test/blob/master/samples/service.py
+
+import time
+import ConfigParser
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+from avocado.utils.service import SpecificServiceManager
+from avocado.utils import distro
+
+
+class service_check(Test):
+
+    def test(self):
+        detected_distro = distro.detect()
+        parser = ConfigParser.ConfigParser()
+        config_file = self.datadir + '/services.cfg'
+        parser.read(config_file)
+        services_list = parser.get(detected_distro.name, 'services').split(',')
```

Comment:
It might be friendlier to catch the `NoSectionError` exception and tell the user what happened (consider this a mild suggestion as the message is sort-of informative enough).